src/ahbicht/expressions/condition_nodes.py

```diff
@@ -78,51 +78,51 @@ def __xor__(self, other):
         return ConditionFulfilledValue.UNFULFILLED
 
 
-@attr.s(auto_attribs=True, kw_only=True)
+@attrs.define(auto_attribs=True, kw_only=True, slots=False)
```

Comment:
What does `slots=False` ?

Reply:
See commit message in https://github.com/Hochfrequenz/ahbicht/pull/106/commits/3ea99d2aba50b604d2b791267bbd84ef4f1d10ad
THe commit messaage also contains further reading links